cli/functionary/utils.py

```diff
@@ -2,19 +2,35 @@
 from rich.table import Table
 
 
-def _get_str_value(value):
-    """Helper function to return a string representation of a value.
+def flatten(results, object_fields):
+    """Flattens any nested objects in results.
 
-    If the passed in value is not a dict, this will return the str() of it.
-    If it is a dict, it will return the str() of the first item that doesn't
-    have a key of 'id', otherwise it will return None.
+    The keys from each dict in results is checked for a replacement
+    in object_fields. If found in object_fields, the original value
+    will be replaced by new entries for each mapping specified.
+
+    Args:
+        results: List of dict objects to process
+        object_fields: mapping of keys from a nested object in each
+            result item that should be in used instead of the object.
```

Comment:
It would be useful to have an example in this docstring of what this actually looks like when you call it.  I don't know that someone just reading the docstring would actually be able to tell what `object_fields` should look like.